### ISSUE TITLE
Remote Follow

### ISSUE THREAD

**Issue by JonLundy** [jointwt/twtxt#349]
It would be cool to have a remote follow link like they have on mastodon servers. If you do not have an account (or are not logged in) on a different pod and are viewing someones profile i would like the follow button to popup a window or text box where you can type in your pod address and it would construct a url to redirect you to your pod external profile page so you can follow them.

<img width="444" alt="image" src="https://user-images.githubusercontent.com/821826/105566547-08857a00-5cea-11eb-9458-fe49fe951df6.png">

**Comment by JonLundy** [comment]
I am aware of viewing someones external profile from my pod to add. I have had happen on more than one occaision where i find someone after clicking on a subject tag finding myself on a different pod and wanting to add someone. in this case i have to copy paste the url and username back at my pod. which feels error prone.

**Comment by prologic** [comment]
Ahh yes! Let's do this! 👌 Any way we can skip a step (entering your home pod url)?

**Comment by JonLundy** [comment]
the external pod would not have any idea who i am since im typicaly not logged in on twtxt.net for example.

**Comment by JonLundy** [comment]
so this is the flow i find .. i find a twt and click the subject to read more. that takes me to a thread on a different pod. and i see someone i want to follow. so i click their name. now because i dont have an account on twtxt.net i cant add them directly. i have to copy thier twtxt link and username and paste them back on my pod.

**Comment by prologic** [comment]
> I am aware of viewing someones external profile from my pod to add. I have had happen on more than one occaision where i find someone after clicking on a subject tag finding myself on a different pod and wanting to add someone. in this case i have to copy paste the url and username back at my pod. which feels error prone.

Pretty much same story for me too, especially now that there are 4 other known pods that aren't my own 🤣

**Comment by prologic** [comment]
> so this is the flow i find .. i find a twt and click the subject to read more. that takes me to a thread on a different pod. and i see someone i want to follow. so i click their name. now because i dont have an account on twtxt.net i cant add them directly. i have to copy thier twtxt link and username and paste them back on my pod.

What if we extend the inter-pod discovery a bit here? Pods _could_ know about the existence of each other and, we _could_ even take advantage of the `Referrer` header in _some_ cases?

**Comment by JonLundy** [comment]
Ah I see so if it gets a referrer it can pre populate the pod.

**Comment by prologic** [comment]
Yeah I'm not sure though I haven't completely thought this through yet I'm still on holidays 🤣

**Comment by prologic** [comment]
So how do we go about implementing this?
